<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GammaRay::PaintBufferViewer</class>
 <widget class="QWidget" name="GammaRay::PaintBufferViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>378</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTreeView" name="commandView">
      <property name="rootIsDecorated">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <widget class="GammaRay::PaintBufferReplayWidget" name="replayWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>182</width>
         <height>372</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GammaRay::PaintBufferReplayWidget</class>
   <extends>QWidget</extends>
   <header>paintbufferreplaywidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
